tox.ini

```diff
@@ -323,7 +323,7 @@ passenv=TEST_*
 
 commands =
 # run only essential tests related to the tracing client
-    tracer: pytest {posargs} --ignore="tests/contrib" --ignore="tests/integration" --ignore="tests/commands" --ignore="tests/opentracer" --ignore="tests/unit" --ignore="tests/internal" tests
+    tracer: pytest {posargs} --ignore="tests/contrib" --ignore="tests/test_integration.py" --ignore="tests/commands" --ignore="tests/opentracer" --ignore="tests/unit" --ignore="tests/internal" tests
```

Comment:
Would it make sense to move to `tests/integration/integration_test.py`?

Doesn't really matter, just a thought.

Reply:
Boarf, having a directory with a single file is not really exciting to me :)

Reply:
BUT WHAT IF WE ADD MORE FILES!!!